src/main/markdown/doc/latest/DevGuideCodingBasicsCompatibility.md

```diff
@@ -1,4 +1,4 @@
-Comptability
+Compatibility
```

Comment:
I don' understand why this (and the modification above in `fragment_merging.md`) ended up here :s